Paper.tex

```diff
@@ -1365,10 +1404,27 @@ \section{Block Finalisation} \label{ch:finalisation}
 The process of finalising a block involves two stages:
 
 \begin{enumerate}
+\item executing withdrawals;
 \item validate transactions;
 \item verify state.
 \end{enumerate}
 
+\subsection{Executing Withdrawals}
+After processing the block's transactions, the withdrawals are executed. A withdrawal is simply an increase of the recipient account's balance of the specified amount. No other balances are decreased, a withdrawal is not a transfer but a creation of funds.
+A withdrawal operation cannot fail and has no gas cost. We define the function $P$ as the withdrawal state transition function:
```

Comment:
FYI, not necessarily an issue, function `P` is already defined in the paper.

Reply:
changes in [55a4099](https://github.com/ethereum/yellowpaper/pull/906/commits/55a4099d6366a2425e4c7c8ad1ba2687cd1abc89)